infra/test/integration/reviewsite/serverspec/default_spec.rb

```diff
@@ -13,9 +13,9 @@
   user    = USER
 end
 
-# Ruby 2.1.6
-describe command('rbenv global 2.1.6 && rbenv version') do
-  its(:stdout) { should match '2.1.6' }
+# Ruby 1.9.3-p448
+describe command('rbenv global 1.9.3-p448 && rbenv version') do
+  its(:stdout) { should match '1.9.3-p448' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.